extensions/card4l-eo/README.md

```diff
@@ -0,0 +1,184 @@
+
+See also the [notes](#notes) regarding the requirements 1.13 and 1.15 for a better understanding of how to use the fields best.
+
+#### Projection
+
+| Field Name                            | Description                                                  | Req.      |
+| ------------------------------------- | ------------------------------------------------------------ | --------- |
+| proj:epsg / proj:wkt2 / proj:projjson | **REQUIRED.** At least one of the three properties `proj:epsg`, `proj:wkt2` or `proj:projjson` is required. It MUST specify the coordinate reference system (1.5) and MAY specify the map projection (1.6). | 1.5 / 1.6 |
+
+#### View
+
+| Field Name           | Description                                                  | Req. |
+| -------------------- | ------------------------------------------------------------ | ---- |
+| view:off_nadir       | **REQUIRED.** The average off-nadir angle, for per-pixel angles, refer to the asset with the key `off-nadir`. Convert to degree, if required. | \*   |
+| view:incidence_angle | **REQUIRED.** The average incidence angle, for per-pixel angles, refer to the asset with the key `incidence-angle`. Convert to degree, if required. | \*   |
+| view:azimuth         | **REQUIRED.** The average azimuth angle, for per-pixel angles, refer to the asset with the key `azimuth`. Convert to degree, if required. | \*   |
+| view:sun_azimuth     | **REQUIRED.** The average sun azimuth angle, for per-pixel angles, refer to the asset with the key `sun-azimuth`. Convert to degree, if required. | \*   |
+| view:sun_elevation   | **REQUIRED.** The average sub elevation angle, for per-pixel angles, refer to the asset with the key `sun-elevation`. Convert to degree, if required. | \*   |
+
+\* = Requirement 2.8 for Surface Temperature (ST) / 2.11 for Surface Reflectance (SR)
+
+### STAC Item Links
+
+| Relation Type            | Description                                                  | Req.                |
+| ------------------------ | ------------------------------------------------------------ | ------------------- |
+| card4l-document          | **REQUIRED.** Provides at least one link to the CARD4L specification document. Word (media type: `application/vnd.openxmlformats-officedocument.wordprocessingml.document`) and/or PDF (media type: `application/pdf`). | *n/a*               |
+| derived_from             | Points back to the source's STAC Item. May be multiple items, if the product is derived from multiple acquisitions. | 1.15                |
+| about                    | Link to algorithms used in the generation process. See also the [notes](#notes) regarding req. 1.13. | 1.13                |
+| related                  | **REQUIRED.** Link to the sources of ancillary or auxiliary data used in the generation process. Excludes DEMs, which use the relation `elevation-model` instead. | 1.14                |
```

Comment:
What does it mean to link to the data source? Is it a URL to a file, or a website landing page indicating how to download the data?
This excludes DEMs, so I think some examples of ancillary data would be useful, e.g., Ozone, Water Vapor

Reply:
Now I see ozone, water vapor, emissivity and others are all their own relation types.  So since this is required I think an example of what this would include should be here.

Reply:
I have no more details as the spec hss and that should be read side by side anyway. Don't want to duplicate all the information given there